services/web/server/src/simcore_service_webserver/projects/projects_handlers_crud.py

```diff
@@ -139,6 +141,8 @@ async def create_projects(request: web.Request):
             query_params=query_params,
             user_id=req_ctx.user_id,
             predefined_project=predefined_project,
+            task_context=jsonable_encoder(req_ctx),
```

Comment:
so here the lrtask context is decided to be exactly as the request context. Is it always the case? 

THOUGHT:
if so, perhaps we need to store the ``request`` (storage) always instead of defining a new ``task_context``... 
AFAIK you want to keep track of the "original" request that triggered the lrtask, right?